easybuild/easyconfigs/b/Buildah/Buildah-1.11.5.eb

```diff
@@ -0,0 +1,38 @@
+easyblock = 'Tarball'
+
+name = 'Buildah'
+version = '1.11.5'
+local_go_version = '1.13.4'
+
+homepage = 'https://buildah.io/'
+description = """Buildah is a tool that facilitates building OCI container images."""
+
+toolchain = SYSTEM
+
+sources = ['go%s.linux-amd64.tar.gz' % local_go_version]
+source_urls = ['https://storage.googleapis.com/golang/']
+checksums = ['692d17071736f74be04a72a06dab9cac1cd759377bd85316e52b2227604c004c']
+
+postinstallcmds = ['mkdir %(builddir)s/go_path && export GOPATH=%(builddir)s/go_path && '
+                   '%(installdir)s/bin/go get github.com/opencontainers/runc && '
+                   '%(installdir)s/bin/go get github.com/containers/buildah ']
+
+# Make-install runc v1.0.0-rc9
+postinstallcmds += [
+    'cd %(builddir)s/go_path/src/github.com/opencontainers/runc && '
+    'export PATH=%(installdir)s/bin/:$PATH && export GOPATH=%(builddir)s/go_path && '
+    'git checkout v1.0.0-rc9 && make BUILDTAGS="" && make install PREFIX=%(installdir)s']
```

Comment:
Does it make sense to define a `local_` var for the open container and set it a the beginning of the recipe, like you did for go?

Reply:
Is it possible to mix `%()s` type strings and the python formated ones?

Reply:
yes. You can pass a dictionary at the end, which makes the mapping...